Samples/SSRSample/src/Server/Server.fs

```diff
@@ -0,0 +1,82 @@
+﻿module Server.Main
+
+open System
+open System.IO
+open System.Threading.Tasks
+open Microsoft.AspNetCore
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.DependencyInjection
+
+open Giraffe
+open Giraffe.GiraffeViewEngine
+open Giraffe.Serialization.Json
+
+open Newtonsoft.Json
+
+open Shared
+
+let clientPath = Path.Combine("..","Client") |> Path.GetFullPath
+let port = 8085us
+
+let initState: Model = {
+  counter = Some 42
+  someString = "Some String"
+  someFloat = 11.11
+  someInt = 22
+}
+
+let getInitCounter () : Task<Model> = task { return initState }
+
+let htmlTemplate =
+  html []
+    [ head [] []
+      body []
+        [ div [_id "elmish-app"] [ rawText (Fable.Helpers.ReactServer.renderToString(Client.View.view initState ignore)) ]
+          script []
+            [ rawText (sprintf """
+            var __INIT_STATE__ = %s
+            """ (toJson initState)) ]
+          script [ _src "http://localhost:8080/public/bundle.js" ] []

```

Comment:
Even if this is a sample, I'd try to avoid the URL literal here. If you can't use `public/bundle.js` because Wepback dev server and Giraffe are running in different ports, I'd use a variable so to minimize the chances `http://localhost:8080` reaches production.